docs/dyn/adexchangebuyer2_v2beta1.accounts.clients.invitations.html

```diff
@@ -105,11 +105,11 @@
     "clientAccountId": "A String", # Numerical account ID of the client buyer
         # that the invited user is associated with.
         # The value of this field is ignored in create operations.
+    "invitationId": "A String", # The unique numerical ID of the invitation that is sent to the user.
```

Comment:
Unrelated:

Just realized this was nondeterministic, how hard would it be to have a fixed ordering?